Active-Directory/Backup-GroupPolicy.ps1

```diff
@@ -63,15 +63,26 @@ param (
 
     [Parameter()]
     [string]
-    $Domain = (Get-WmiObject Win32_ComputerSystem).Domain,
-
-    [Parameter()]
-    [string]
+    $Domain = $env:USERDNSDOMAIN,
```

Comment:
This will pull the current domain from the system environment variables instead of processing a WMI query.